charts/timescaledb-single/templates/job-update-patroni.yaml

```diff
@@ -37,6 +37,10 @@ spec:
         heritage: {{ .Release.Service }}
     spec:
       restartPolicy: OnFailure
+    {{- with .Values.tolerations }}
+      tolerations:
+{{ toYaml . | indent 8 }}
+    {{- end }}
```

Comment:
```suggestion
      tolerations:
        {{ toYaml .Values.tolerations | nindent 8 }}
```

Reply:
Now that I think about it, what happens when the list is empty? Shouldn't this block guarded against that?